templates/profile/prometheus/exporter/mysql/target.yaml.erb

```diff
@@ -1,5 +1,5 @@
-- targets: [ '<%= @ipaddress %>:9101' ]
+- targets: [ '<%= @ipaddress %>:9100' ]
```

Comment:
Does the mysql exporter really use 9100? If so, we'll need to not use the default, because this port is already taken by the node exporter.

Reply:
Ah, good catch.  I wasn't making that connection.   Port is actually 9104.  Will push with my next update.